src/lib/y2storage/disk.rb

```diff
@@ -68,6 +68,13 @@ def usb?
       transport.to_sym == :usb
     end
 
+    # Checks if it's in network
+    #
+    # @return [Boolean]
+    def network_trasporter?
+      transport.to_sym == :iscsi or transport.to_sym == :fcoe
```

Comment:
I would expect this comparison to be in the `DataTransport` class. So here you would do:
```ruby
transport.network?
```
Instead of comparing here for all the possible values.